babel.config.js

```diff
@@ -12,17 +12,17 @@ module.exports = api => {
 
     ignore,
 
-    presets: ['@babel/preset-react', '@babel/preset-env'],
+    presets: ['@babel/preset-react', !esm && '@babel/preset-env'].filter(Boolean),
```

Comment:
i'm looking at this line and it looks like it's always including preset-env....am i reading it wrong?

Reply:
preset-env is only included when esm is not true. When the esm build runs esm is true though I should probably make sure it's false when the CJS build runs.

Reply:
nevermind - i can't read